backend/organization/views/organization_views.py

```diff
@@ -84,15 +84,23 @@ def get_serializer_context(self):
         return {"language_code": self.request.LANGUAGE_CODE}
 
     def get_queryset(self):
-        organizations = Organization.objects.all()
+        organizations = Organization.objects.all().prefetch_related(
+            Prefetch(
+                'tag_organization',
+                queryset=OrganizationTagging.objects.select_related(
+                    'organization_tag'
+                )
+            ),
+            'organization_member'
+        ).select_related('language', 'location')
```

Comment:
Should we make this dependent on which query_params were passed? If I understand this correctly, this PR would significantly improve query performance for queries filtering by hub or location but would do unnecessary additional work when simply loading the browse page without filters.

Reply:
Generally this can be extremely useful though, e.g. for the project page

Reply:
Discussed offline, these tables are used in serializer so it will be helpful to prefetch them with or without filters.